### PR TITLE
fix generation of path to rebar.config

### PR DESCRIPTION
**Pull request by JakaBac**
URLs from `module.getContentRootsList().getUrls()` come in the following form `file://X:/Devel/testing/test` (since they are defined as  `<content url="file://$MODULE_DIR$">` in the iml file)
But java.net.URL expects URLs with drive letters with an extra `'/'` like `file:///X:/Devel/testing/test`

`(new URL("file:///X:/Devel/testing/test").getPath()` returns `X:\Devel\testing\test`
but
`(new URL("file://X:/Devel/testing/test").getPath()` returns `\Devel\testing\test`

This makes the drive letter of the module's rebar.config dependant on the installation directory of IntelliJ.
If both the project and IntelliJ are on the same drive, everything works as expected, otherwise the plugin will look for rebar.config in a completely wrong location (the drive where IntelliJ is installed) and error with "rebar.config not found" message.

**TL;DR**
Fixes locating problems locating rebar.config on Windows

### CODE REVIEW

**Comment by ignatov** [comment]
@JakaBac many thanks!

**Comment by ignatov** [comment]
I've uploaded https://plugins.jetbrains.com/plugin/7083-erlang/update/44618
